config.example.yaml

```diff
@@ -57,5 +57,5 @@ council_changelog: 1234567890
 required_difference: 5  # Majority required for this blob to move into the next stage
 required_votes: 15  # Minimum amount of total votes before moving to next stage
 
-# Versus type
+# Versus voting message type, Verbose (true) or Compact (false)
```

Comment:
Still not satisfied with this description. Config descriptions should explain their effect, not specify their label.